documentation/DEPLOYMENT-AZD.md

```diff
@@ -175,9 +174,9 @@ As you can see from the first line of the file, the manifest is backed by a lang
 
 ## Infrastructure as Code - alternatives
```

Comment:
"alternatives" in the titel sounds as if there is an additional default path. How about dropping it? And maybe a sentence on bicep being default for azd.

Reply:
Fixed with commit <https://github.com/Azure-Samples/app-service-javascript-sap-cloud-sdk-quickstart/commit/c59ed5bcf790245e421476d477e5e3ebf1566450>